### PR TITLE
Fix selector for kube_node_status_allocatable_memory_bytes

### PR DESCRIPTION
**Pull request by jacknagel**

### CODE REVIEW

File: alerts/resource_alerts.libsonnet [hunk 1]
```diff
@@ -82,7 +82,7 @@
             expr: |||
               sum(kube_resourcequota{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, type="hard", resource="memory"})
                 /
-              sum(kube_node_status_allocatable_memory_bytes{%(nodeExporterSelector)s})
+              sum(kube_node_status_allocatable_memory_bytes{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s})
```
Comment (brancz) [inline alerts/resource_alerts.libsonnet:85]: `%(prefixedNamespaceSelector)s` only applies to namespaced APIs, which the node API is not, so we don't need that part here.

**Review by brancz — COMMENTED** [review]
Thanks for catching this! Just one small thing, otherwise lgtm.

File: alerts/resource_alerts.libsonnet (hunk 1 above)
Comment (jacknagel) [reply alerts/resource_alerts.libsonnet]: Fixed, thanks.

**Comment by brancz** [comment]
Thank you!

**Pull request merged**